locales/app.yml

```
@@ -143,658 +165,764 @@ _version: 2
   fr: "Voulez-vous l'installer ?"
   zh_CN: "是否立即安装？"
   zh_TW: "是否現在安裝？"
+  de: "Möchten Sie sie installieren?"
 "No global packages installed":
   en: "No global packages installed"
   es: "No hay paquetes globales instalados"
   fr: "Aucun paquet global n'est installé"
   zh_CN: "尚未安装全局软件包"
   zh_TW: "尚未安裝全域套件"
+  de: "Keine globalen Pakete installiert"
 "Remote Topgrade launched in Tmux":
   en: "Remote Topgrade launched in Tmux"
   es: "Topgrade remoto lanzado en Tmux"
   fr: "Topgrade distant lancé dans Tmux"
   zh_CN: "已在 Tmux 中启动远程 Topgrade"
   zh_TW: "已在 Tmux 中啟動遠端 Topgrade 程序"
+  de: "Remote-Topgrade in Tmux gestartet"
 "Remote Topgrade launched in an external terminal":
   en: "Remote Topgrade launched in an external terminal"
   es: "Topgrade remoto iniciado en una terminal externa"
   fr: "Topgrade distant lancé dans un terminal externe"
   zh_CN: "已在新终端窗口中启动远程 Topgrade"
   zh_TW: "已在新終端機視窗中啟動遠端 Topgrade"
+  de: "Remote-Topgrade in einem externen Terminal gestartet"
 "Collecting Vagrant boxes":
   en: "Collecting Vagrant boxes"
   es: "Recolectando cajas Vagrant"
   fr: "Collecte des boîtes Vagrant"
   zh_CN: "正在收集 Vagrant 容器"
   zh_TW: "正在蒐集 Vagrant 容器"
+  de: "Sammle Vagrant-Boxen"
 "No Vagrant directories were specified in the configuration file":
   en: "No Vagrant directories were specified in the configuration file"
   es: "No se especificaron directorios Vagrant en el archivo de configuración"
   fr: "Aucun répertoire Vagrant n'est spécifié dans le fichier de configuration"
   zh_CN: "尚未在配置中指定 Vagrant 目录"
   zh_TW: "尚未在設定檔中指定 Vagrant 資料夾"
+  de: "In der Konfigurationsdatei wurden keine Vagrant-Verzeichnisse angegeben"
 "Vagrant boxes":
   en: "Vagrant boxes"
   es: "Cajas Vagrant"
   fr: "Boîtes Vagrant"
   zh_CN: "Vagrant 容器"
   zh_TW: "Vagrant 容器"
+  de: "Vagrant-Boxen"
 "No outdated boxes":
   en: "No outdated boxes"
   es: "Sin cajas obsoletas"
   fr: "Aucune boîte obsolète"
   zh_CN: "没有需要更新的容器"
   zh_TW: "未有需要更新的容器"
+  de: "Keine veralteten Boxen"
 "Summary":
   en: "Summary"
   es: "Resumen"
   fr: "Résumé"
   zh_CN: "结果"
   zh_TW: "結果"
+  de: "Zusammenfassung"
 "Topgrade finished with errors":
   en: "Topgrade finished with errors"
   es: "Topgrade finalizado con errores"
   fr: "Topgrade terminé avec des erreurs"
   zh_CN: "Topgrade 运行完成，但存在错误"
   zh_TW: "Topgrade 執行部分成功"
+  de: "Topgrade wurde mit Fehlern beendet"
 "Topgrade finished successfully":
   en: "Topgrade finished successfully"
   es: "Topgrade finalizó exitosamente"
   fr: "Topgrade terminé avec succès"
   zh_CN: "Topgrade 运行成功"
   zh_TW: "Topgrade 執行成功"
+  de: "Topgrade wurde erfolgreich beendet"
 "Topgrade {version_str} Breaking Changes":
   en: "Topgrade %{version_str} Breaking Changes"
   es: "Topgrade %{version_str} Cambios Importantes"
   fr: "Topgrade %{version_str} Changements Cassants"
   zh_CN: "Topgrade %{version_str} 破坏性更改"
   zh_TW: "Topgrade %{version_str} 重大更改"
+  de: "Topgrade %{version_str} Breaking Changes"
```

Looks like you forgot to translate this:

```suggestion
  de: "Topgrade %{version_str} Inkompatible Änderungen"
```

---

Should this be translated?

---

I totally agree, this should definitely be translated!